File: Java Fundamentals/Data types and variables/Lab Tasks/Task2_PoundsToDollars.java
import java.util.Scanner;

public class Task2_PoundsToDollars {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        double pounds = Double.parseDouble(scanner.nextLine());

        double dollars = pounds * 1.31;

        System.out.printf("%.3f", dollars);

//1 pound = 1.31 dollars

    }
}

File: C#/C# Fundamentals/Data Type & Variables/Lab Tasks/Task2_PoundsToDollars.cs
using System;

namespace Task2_PoundsToDollars
{
    class Task2_PoundsToDollars
    {
        static void Main(string[] args)
        {
            double pounds  = double.Parse(Console.ReadLine());
            double dollars = pounds * 1.31;

            Console.WriteLine($"{dollars:f3}");
        }
    }
}
